Frontend/app/styles/_app.scss

```diff
@@ -0,0 +1,11 @@
+.bc-c-title {
+  border-bottom: 2px solid $color_nhsuk-blue;
```

Comment:
if these are duplicated then move to BCL

Reply:
I need to remove these. They are not being used.